components/components-definition.json

```diff
@@ -204,7 +204,7 @@
             "label": "{{ 'DESIGN_COMPONENT_SOCIAL_MEDIA_LABEL' | translate }}",
             "selectionMethod": "handle",
             "properties": [
-                "media-properties:media",
+                "media-properties-social:media",
```

Comment:
Since we're separating the two, why not call them `social-media-properties` and `video-properties`?

Reply:
With the changes for slideshow properties, you can do it like:
```{ "name": "media-properties", "control": { "mediaType": "social" }, "directiveKey": "media" }```

Then it's merged with the property from componentProperties.

Reply:
ok